IMAGES.rst

```diff
@@ -320,10 +320,10 @@ Images built as "Run ID snapshot":
 
 .. code-block:: bash
 
-  docker.pkg.github.com.io/apache-airflow/<BRANCH>-pythonX.Y-ci-v2:<RUNID>    - for CI images
-  docker.pkg.github.com/apache-airflow/<BRANCH>-pythonX.Y-v2:<RUNID>       - for production images
-  docker.pkg.github.com/apache-airflow/<BRANCH>-pythonX.Y-build-v2:<RUNID> - for production build stage
-  docker.pkg.github.com/apache-airflow/pythonX.Y-<BRANCH>-v2:X.Y-slim-buster-<RUN_ID>  - for base Python images
+  docker.pkg.github.com.io/apache-airflow/<BRANCH>-pythonX.Y-ci-v2:<RUN_ID>    - for CI images
+  docker.pkg.github.com/apache-airflow/<BRANCH>-pythonX.Y-v2:<RUN_ID>       - for production images
+  docker.pkg.github.com/apache-airflow/<BRANCH>-pythonX.Y-build-v2:<RUN_ID> - for production build stage
+  docker.pkg.github.com/apache-airflow/python-v2:X.Y-slim-buster-<RUN_ID>  - for base Python images
```

Comment:
Are these changes simply cleaning/correcting? Or they are related to the main change in this PR?

Reply:
They are correcting. The names were wrong (I found it while preparing the script). Happy to split it off if needed

Reply:
Personally I'm ok to keep them here, but good to update the commit msg and PR title/description as well. Do you agree?

Reply:
Absolutely - you are perfectly right @XD-DENG. I've added all the explanation as the commit message

Reply:
And added it as comment in the script.

Indeed. Assumptions (which I have and others don't) are the root of evil :)